scicamera/camera.py

```diff
@@ -34,6 +34,14 @@
 class CameraManager:
     cameras: Dict[int, Camera]
```

Comment:
It really seems like this should be a private variable since the `get_camera` and `add` functions manage most of the dict-things.